### DISCUSSION TITLE
'Not found' errors for login page, and any url that includes /livehelperchat/lhc_web/index.php/...

### DISCUSSION THREAD

**Discussion by jamminjames** [LiveHelperChat/livehelperchat#2032] [General]
The LiveHelperChat widget is no longer showing on the page, and I am getting "Not found" errors for the login page and anything coming after /index.php/ in the url. For example, in the browser development tools console, for
`GET https://subs.humortimes.com/livehelperchat/lhc_web/index.php/widgetrestapi/settings?` (bunch of settings after this), I get a 404 not found error.

For `/livehelperchat/lhc_web/design/defaulttheme/js/widgetv2/index.js?20231128`, I get: 
```
Uncaught SyntaxError: Unexpected token '<', "<html>
<he"... is not valid JSON
livehelperchat/lhc_web/index.php/widgetrestapi/settings? ...
```

With debug on, when I go to `/livehelperchat/lhc_web/index.php`, there are a lot of lines like:
```
0: erLhcoreClassDesign::designtpl - lhchat/start.tpl.php | 2023-12-28 22:12:22 -0800
Not found IN - /var/www/subs.humortimes/livehelperchat/lhc_web/design/customtheme/tpl/lhchat/start.tpl.php
Found IN - /var/www/subs.humortimes/livehelperchat/lhc_web/design/defaulttheme/tpl/lhchat/start.tpl.php
0: erLhcoreClassDesign::design - js/widgetv2 | 2023-12-28 22:12:22 -0800
Not found IN - /var/www/subs.humortimes/livehelperchat/lhc_web/design/customtheme/js/widgetv2
Found IN - /var/www/subs.humortimes/livehelperchat/lhc_web/design/defaulttheme/js/widgetv2
```
... not sure if that's relevant. 

I tried turning off the cache in `/livehelperchat/lhc_web/settings/settings.ini.php`, as suggested elsewhere, and turning debugging on. The `/livehelperchat/lhc_web/cache/default.log` stays blank. 

The /livehelperchat/ folders all have the correct ownership and chmod settings.

What should I do? Thanks for any help.

**Comment by remdex** [comment]
You could try to read this article - https://doc.livehelperchat.com/docs/nginx-configuration-tips there is sample for subdirectory support. Or you can try to run it under it's own subdomain.

**Reply by jamminjames** [reply]
Great, thanks! I just needed to add the Nginx location block for the lhc_directory (in my case 'livehelperchat'), like:

```
location /livehelperchat {
    try_files $uri $uri/ /livehelperchat/lhc_web/index.php?q=$uri&$args;
}
```
